samples/tfm_integration/psa_level_1/Kconfig

```diff
@@ -19,4 +19,19 @@ config PSA_SHELL
 	help
 	  Enabling this option will make the 'psa' shell command available.
 
+choice
+	prompt "Private Key"
+
+config PRIVATE_KEY_STATIC
+	bool "Static"
+	help
+	  A static key value will be used for the private key.
+
+config PRIVATE_KEY_RANDOM
```

Comment:
I would set the random key as the default here, as it is the safest option

Reply:
@frkv @microbuilder 
The config `PRIVATE_KEY_RANDOM` is not used in the code. When config `PRIVATE_KEY_STATIC` is enabled, a static key is used otherwise the key data is generated using `psa_generate_random`. By default, `PRIVATE_KEY_STATIC` is disabled, therefore random key data. I'd propose to remove the config option `PRIVATE_KEY_RANDOM`.

Reply:
Sorry, I misunderstood how `choice` config option worked. Reading https://docs.zephyrproject.org/latest/guides/kconfig/setting.html#configuring-choices made it clear. I'll add `PRIVATE_KEY_RANDOM` as the default choice.